browser-switch/src/main/java/com/braintreepayments/api/ChromeCustomTabsInternalClient.java

```diff
@@ -20,7 +21,7 @@ class ChromeCustomTabsInternalClient {
         this.customTabsIntentBuilder = builder;
     }
 
-    void launchUrl(Context context, Uri url, boolean launchAsNewTask) {
+    void launchUrl(Context context, Uri url, boolean launchAsNewTask) throws ActivityNotFoundException{
```

Comment:
This throws an exception if no browser is installed - ChromeCustomTabs intent just wraps plain browser intent so there's actually no need for a separate check for CCT capability - it will just fall back to non-CCT browser, or throw this exception if there's no browser at all.